ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/picklist/autocompleteActivityPicklistQuestion.component.ts

```diff
@@ -169,4 +180,10 @@ export class AutocompleteActivityPicklistQuestion extends BaseActivityPicklistQu
     displayAutoComplete(option: ActivityPicklistOption | string): string {
         return typeof option === 'string' ? option : (option?.optionLabel || '');
     }
+
+    @HostListener('window: scroll') public onWindowScroll(): void {
+        if (this.autoComplete?.panelOpen) {
+            this.autoComplete.closePanel();
```

Comment:
how about using updatePosition api (https://material.angular.io/components/autocomplete/api) instead of closing the popup?

Reply:
It does not help in our case.

Reply:
Agreed to keep the autocomplete dropdown under the header by scrolling.